### ISSUE TITLE
is it posssible to add search functionality with select

### ISSUE THREAD

**Issue by NitinMagdum** [HaidarZ/ng-mat-select-infinite-scroll#11]
is it posssible to add search functionality with select

**Comment by NitinMagdum** [comment]
i added both ngx-mat-select-search and ng-mat-select-infinite-scroll
https://stackblitz.com/edit/ng-mat-select-infinite-scroll-3gsd97?file=src%2Fapp%2Fapp.component.ts
but here the issue i am facing is after Option 9 any option i select it is not getting set to the select input box

**Comment by HaidarZ** [comment]
Hello Sur,
Sorry for the late resposne , would you please inform me if you solved your problem .

**Comment by NitinMagdum** [comment]
No, the issue is not resolved it would be great if working example provided

**Comment by NitinMagdum** [comment]
this plugin is not working in ie

**Comment by Neizan93** [comment]
Same issue here
